src/utils/save_game/save_game.h

```diff
@@ -0,0 +1,12 @@
+#include "../../creature/creature.h"
+#include "../../creature/main_character/main_character.h"
+#include <iostream>
+#include <vector>
+
+#pragma once
+/**
```

Comment:
Comments should have no separating lines between them and that which they explain.

Also, you have two functions here, but only one comment. Please add a comment to the second function as well.